tests/client/test_client_key_meilisearch.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from tests import common
+from datetime import datetime, timedelta
+
+def test_get_keys_default(client):
+    """Tests if public and private keys have been generated and can be retrieved."""
+    key = client.get_keys()
+    assert isinstance(key, list)
+    assert len(key) >= 2
```

Comment:
If this is the default the number of keys should always be exactly 2 and never greater.

Reply:
I always have a doubt on the fact that the tests are executed in order and I forget if according to the SDK if they are all independent.